indexbuilder.go

```diff
@@ -200,6 +193,14 @@ type IndexBuilder struct {
 
 	// a sortable 20 chars long id.
 	ID string
+
+	// The version we will write to disk. Sourcegraph Specific. This is to
+	// enable feature flagging new format versions.
+	indexFormatVersion int
+	featureVersion     int
+
+	symID     uint32
+	symKindID uint32
```

Comment:
this read nicer when the symbol stuff was all together. Likely a candidate for all the sym stuff to be in its own struct to be honest.